news.md

```diff
@@ -0,0 +1,27 @@
+---
+title: News
+layout: default
```

Comment:
We should use a consistent layout template (like the one used for the other pages like [about](https://github.com/sjsrey/pysal.github.io/blob/jekyll/about.md) and [Documentation](https://github.com/sjsrey/pysal.github.io/blob/jekyll/documentation.md)). Therefore, it should be `layout:page_md`.

Reply:
will fix.